scripts/uploadOffchainMetadata.ts

```diff
@@ -36,14 +36,14 @@ const payer = provider.wallet["payer"];
 const umi = createUmi(provider.connection);
 umi.use(keypairIdentity(payer));
 
-type AcceptedVaultTokens = "META" | "USDC";
-type ConditionalToken = "pMETA" | "fMETA" | "pUSDC" | "fUSDC";
+type AcceptedVaultTokens = "META" | "USDC" | "FUTURE" | "TOKE" | "DEAN";
```

Comment:
Ideally we have a better or shared lib or SOMETHING else other than adding these.